graphql/src/main/java/io/micronaut/configuration/graphql/ws/apollo/GraphQLApolloWsConfiguration.java

```diff
@@ -97,6 +108,15 @@ public String getPath() {
         return path;
     }
 
+    /**
+     * Sets the GraphQL websocket path.
```

Comment:
make sure the javadoc which goes into the configuration reference documentation outputs the default value